sdk-actors/src/main/java/io/dapr/actors/ActorMethod.java

```diff
@@ -21,5 +21,12 @@
    *
    * @return Actor's method return type.
    */
-  Class returns();
+  Class returns() default Void.class;

```

Comment:
To make this optional, a default value must be provided. On the other hand, `null` is not allowed in Java for annotation attributes. The best I found was to use `Void.class` here. The idea is that this is only needed when the result object is Mono<T>, so T can be hinted to the Actor runtime - this is a limitation of the Java programming language that we need to overcome.

Reply:
The only other option I could think of would be to declare your own sentinel value.

I'm fine either way.

Reply:
I am leaning towards using a sentinel class, called `Default` for that.

Reply:
Created an "internal" class called "Undefined". It reads better now. Please, take a look.